src/index.ts

```diff
@@ -79,6 +79,9 @@ async function setDebugSession(
     debug.session.client = client;
   }
   await debug.session.restoreState();
+  if (debug.canStart()) {
```

Comment:
Wondering if this shouldn't be before `restoreState` (not sure yet).

Reply:
I don't think so because you might restore a session that already has a running debugger.